compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -671,6 +672,16 @@ impl<'hir> Map<'hir> {
         }
     }
 
+    pub fn visit_module_items<V>(self, visitor: &mut V)
+    where
+        V: ModuleVisitor,
```

Comment:
This method is identical to `for_each_module`. Please prefer using it.

Reply:
This change did not actually see any perf improvements so I will revert this specific change since I see the `ItemLikeVisitor` is being actively cleaned up in another PR.